rules/no-truthy-falsy.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const {
+  getDocsUrl,
+  expectCase,
+  expectNotCase,
+  expectResolveCase,
+  expectRejectCase,
+  method,
+} = require('./util');
+
+module.exports = {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          !(
+            expectNotCase(node) ||
+            expectResolveCase(node) ||
+            expectRejectCase(node)
+          ) &&
+          expectCase(node)
+        ) {
+          if (method(node).name === 'toBeTruthy') {
+            context.report({
+              message: 'Use toBe(true) instead',
```

Comment:
I don't think the message should be `toBe(true)`, as it might be more correct to assert `not.toBeNull()` etc. Just saying "Avoid using `toBeTruthy`" or something seems better to me 🙂